api/v2.0/swagger.yaml

```diff
@@ -8657,3 +8756,71 @@ definitions:
         format: int64
         description: The total storage consumption of blobs, only be seen by the system admin
         x-omitempty: false
+  ScanDataExportCriteria:
+    type: object
+    description: The criteria to select the scan data to export.
+    properties:
+      projects:
+        type: array
+        items:
+          type: integer
+          format: int64
+        description: A list of one or more projects for which to export the scan data, defaults to all if empty
+      labels:
+        type: array
+        items:
+          type: integer
+          format: int64
+        description: A list of one or more labels for which to export the scan data, defaults to all if empty
+      repositories:
+        type: array
+        items:
+          type: integer
+          format: int64
+        description: A list of repositories for which to export the scan data, defaults to all if empty
+      cveIds:
+        type: array
+        items:
+          type: string
+      tags:
```

Comment:
Should  use  double star rule  for "repos" and "tags".
And for "cveIds", need to discuss